nautobot/core/tests/test_checks.py

```diff
@@ -42,3 +42,18 @@ def test_check_release_check_url(self):
     def test_check_storage_config_and_backend(self):
         """Warn if STORAGE_CONFIG and STORAGE_BACKEND aren't mutually set."""
         self.assertEqual(checks.check_storage_config_and_backend(None), [checks.W005])
+
+    @override_settings(
+        BASE_PATH="nautobot/",
+    )
+    def test_check_base_path_and_login_url_failure(self):
+        """Error if BASE_PATH and LOGIN_URL aren't mutually set."""
+        self.assertEqual(checks.check_base_path_and_login_url(None), [checks.E006])
+
+    @override_settings(
+        BASE_PATH="nautobot/",
+        LOGIN_URL="/nautobot/login/",
```

Comment:
Just checking, as I see this below too - does `LOGIN_URL` *have to* start with a leading `/` and end with a trailing `/`? Does `BASE_PATH` need to *not* start with a leading `/`? Should these be additional checks, if so?

Reply:
`LOGIN_URL` has to be a valid URL. All URLs end in slashes so if it's not there Django will redirect because of [`APPEND_SLASH`](https://docs.djangoproject.com/en/3.1/ref/settings/#append-slash) (which defaults to `True`)

Base on the way it's written currently, `BASE_PATH` should not start with a `/`. This isn't built-in behavior and happens inside of `nautobot.core.urls`.